src/engine/controls/cuecontrol.cpp

```diff
@@ -223,7 +223,7 @@ CueControl::CueControl(QString group,
     m_pVinylControlMode = new ControlProxy(group, "vinylcontrol_mode");
 
     m_pHotcueFocus = new ControlObject(ConfigKey(group, "hotcue_focus"));
-    m_pHotcueFocus->set(Cue::kNoHotCue);
+    m_pHotcueFocus->set(Cue::kNoHotCue + 1);
```

Comment:
- Please define a constant `kFirstHotCue`. Don't make any assumptions that `kNoHotCue`always precedes the first valid number, it could be any number that is not a valid hot cue number.
- Is the intention here to always focus the first hot cue by default?